Realm/Realm/RealmConfigurationBase.cs

```diff
@@ -45,6 +45,8 @@ public abstract class RealmConfigurationBase
 
         internal bool Dynamic;
 
+        internal virtual bool EnableCache => true;
```

Comment:
I think we're better off with making this a simple field, like `Dynamic, not a virtual property.

Reply:
Then we'll have to set it which is annoying :P

Reply:
Like virtual dispatch is much better :P

Reply:
I don't see it => it doesn't exist ;) Okay, okay... 😩